eq-author/src/App/QuestionnaireDesignPage/MainNavigation/index.test.js

```diff
@@ -68,15 +68,13 @@ describe("MainNavigation", () => {
 
     const viewSurveyBtn = getByTestId("btn-preview");
     const settingsBtn = getByTestId("btn-settings");
-    const sharingBtn = getByTestId("btn-share");
     const historyBtn = getByTestId("btn-history");
     const metadataBtn = getByTestId("btn-metadata");
     const qcodesBtn = getByTestId("btn-qcodes");
     const publishBtn = getByTestId("btn-publish");
 
     expect(viewSurveyBtn).not.toBeDisabled();
     expect(settingsBtn).not.toBeDisabled();
-    expect(sharingBtn).not.toBeDisabled();
```

Comment:
You've replaced the share modal Button with a RouteButton - Should still include a test to check share RouteButton exists / not disabled ?
getByTestId("btn-sharing");